syntax = "proto3";

message Person {
    int32 id = 1;
    string first_name = 2;
    string last_name  = 3;
    string company_name = 4;
}

message Empty {

}


service PersonRequest {
    rpc PersonLookup (Empty) returns (stream Person);
}